command/import.go

```diff
@@ -307,9 +307,9 @@ Options:
 
   -allow-missing-config   Allow import when no resource configuration block exists.
 
-  -input=true             Ask for input for variables if not directly set.
+  -input=false            Disable interactive input prompts.
 
-  -lock=true              Lock the state file when locking is supported.
+  -lock                   Lock the state file when locking is supported.
```

Comment:
If I'm reading the code correctly, I think `-lock` defaults to true. Perhaps we want to specify `-lock=false` as disabling locking here (and throughout)?

Reply:
Hmm yes, you are right in fact. I was confused by the very thing that this editing was intended to help with!

I'll change all of these to say `-lock=false`. Since I've been piling on to `terraform state mv` and `terraform state rm` in later commits here I'll also fix those up a little while I'm here, although I'm going to keep that relatively light for now because it's already quite off the path of what I intended to do with this PR in the first place.